### PR TITLE
Fix docstrings YOLO node

### PR DESCRIPTION
**Pull request by JeromeWYCB**
Added docstrings for yolo node.

### CODE REVIEW

File: peekingduck/pipeline/nodes/model/yolo.py [hunk 1]
```diff
@@ -23,7 +23,72 @@
...
+
+            image resolution passed to the YOLO model.
+
+        num_classes (:obj:`int`): **default = 80 **
+
+            maximum number of objects to be detected.
+
+        detect_ids (:obj:`int`): **default = 0 **
+
+            list of object class ids to be detected.
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/model/yolo.py:65]: is this a list or an int?

Comment (sidney-tio) [inline peekingduck/pipeline/nodes/model/yolo.py:65]: I have also added the YOLO Class ID reference under `glossary.rst`. Add the link to it. 

See for reference: https://github.com/aimakerspace/PeekingDuck/blob/101760898e8efc1c42d880e86ebd97a23dee888b/peekingduck/pipeline/nodes/model/posenet.py#L30

Comment (JeromeWYCB) [reply peekingduck/pipeline/nodes/model/yolo.py]: Sorry, overlooked. This is a list of ids.

Comment (JeromeWYCB) [reply peekingduck/pipeline/nodes/model/yolo.py]: I have added the link to it.

**Pull request merged**